app/services/txt2sql_engine.py

```diff
@@ -324,4 +324,22 @@ def txt2sql_engine(self,user,user_prompt,thread_id):
         else:
             response={"result":explanation}
             result={'success':False,'message':explanation,'data':[response]} 
-        return result
\ No newline at end of file
+        return result
+    
+    def get_table_columns(self, user, table_name):
+        """
+        Returns the list of columns for a given table in the user's database schema.
+        """
```

Comment:
```pythondef get_table_columns(self, user, table_name):
    """
    Returns the list of columns for a given table in the user's database schema.
    
    Args:
        user: User object/tuple containing user information
        table_name (str): Name of the table to get columns for
        
    Returns:
        list: List of column names for the specified table
    """``` ``criticality: MINOR `` 

 > Consider improving the docstring to include parameter types and return type information for better code documentation.